packages/flutter_tools/lib/src/android/gradle.dart

```diff
@@ -806,14 +807,13 @@ class AndroidGradleBuilder implements AndroidBuilder {
 /// Prints how to consume the AAR from a host app.
 void printHowToConsumeAar({
   @required Set<String> buildModes,
-  @required String androidPackage,
+  String androidPackage = 'unknown',
```

Comment:
It's possible to pass a nullable `androidPackage` here.  Since it's just used for informational printing, default to "unknown".